Building native image of camel-java fails with maven plugin surefire version issue

quarkus-quickstarts/camel-java (master)/$ mvn clean package -Pnative
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-failsafe-plugin must be a valid version but is '${surefire.version}'. @ line 95, column 22
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.acme:camel-java:1.0-SNAPSHOT (/Users/davsclaus/workspace/quarkus-quickstarts/camel-java/pom.xml) has 1 error
[ERROR]     'build.plugins.plugin.version' for org.apache.maven.plugins:maven-failsafe-plugin must be a valid version but is '${surefire.version}'. @ line 95, column 22
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException

--- a/camel-java/pom.xml
+++ b/camel-java/pom.xml
@@ -92,7 +92,7 @@
           </plugin>
           <plugin>
             <artifactId>maven-failsafe-plugin</artifactId>
-            <version>${surefire.version}</version>
+            <version>${surefire-plugin.version}</version>
             <executions>
               <execution>
                 <goals>

